app/components/input-field.js

```diff
@@ -0,0 +1,33 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class InputFieldComponent extends Component {
+  @tracked inputValue = null;
```

Comment:
This should not be on the class, as it's only used within the `handleInputFieldChange` function.

Reply:
Done